packages/slice-machine/postinstall.js

```diff
@@ -57,11 +57,11 @@ function installSMScript(pkg) {
   const smModuleCWD = require.main.paths[0].split("node_modules")[0];
   const { pkg, smConfig } = retrieveConfigFiles(projectCWD);
 
-  if (pkg && smConfig) {
-    installSMScript(pkg);
-    writeSMVersion(smModuleCWD, smConfig);
-    return;
-  }
-  if (!pkg) console.error("Missing file package.json");
-  if (!smConfig) console.error("Missing file sm.json");
+  if (pkg.value) installSMScript(pkg);
```

Comment:
what happen if pkg is not defined ? With this code it will crash trying to call value on undefined ?

Reply:
The property pkg already exist, it's the value inside that might be null if the file doesn't exist https://github.com/prismicio/slice-machine/blob/c690dfb5c65842be50540294b3a664eb06502a8a/packages/slice-machine/postinstall.js#L16

Reply:
In the older version of the code it's possible for `console.error` to be called twice.
is that the behaviour we want to remove?